Lil-Peep/src/styles/modules/related-products.scss

```diff
@@ -0,0 +1,73 @@
+.related-product-container {
+  background-color: #{$product-card-category};
```

Comment:
This also works like `background-color: $product-card-category`, it's cleaner and easier to read.